messages/0.1.5.txt

```diff
@@ -0,0 +1,16 @@
+Welcome to the TypeScript plugin for Sublime Text!
+
+Details about this package and its feature can be found on its GitHub
+repository at https://github.com/Microsoft/TypeScript-Sublime-Plugin.
+
+v0.1.5 updates:
+	- Code base refactored for easier future maintenance
+	- Fixed several issues of the syntax definition file
+	- Fixed performance issues caused by quickinfo requests on large files
+	- Fixed the bug that "}" and ";" won't show up when the formatOnKey command fails
+	- Fixed wrong indentation between curly braces
+	- Fixed issues with signature tooltip positioning
+	- Fixed incorrect error spans for Sublime Text 2
+	- Resolved confusion caused by certain snippet triggers in the auto completion member list 
+	- Added build function for configured projects (with tsconfig.json) and loose files. TypeScript files can now be built via "^+B"
```

Comment:
This is actually Cmd+B on a Mac I believe.  Is this description Windows specific?